client/src/components/UIElements/Button.tsx

```diff
@@ -0,0 +1,19 @@
+import "./Button.scss";
+
+type buttonProps = {
+  onClick?: () => void;
+  children: string;
+  inverse?: boolean;
+  bold?: boolean;
+};
+
+const Button = (props: buttonProps) => {
+
+  return (
+    <button className={`button ${props.inverse && 'inverse'} ${props.bold && 'bold'}`} onClick={props.onClick}>
```

Comment:
checkout classNames lib for multiple class names. You might find it more readable.
https://www.npmjs.com/package/classnames